ocs_ci/deployment/provider_client/storage_client_deployment.py

```diff
@@ -0,0 +1,630 @@
+
+        resource_name = network_policy_data["metadata"]["name"]
+
+        # Check network policy for the namespace_to_create_storage_client is available or not
+        network_policy_obj = ocp.OCP(
+            kind="NetworkPolicy", namespace=namespace_to_create_storage_client
+        )
+
+        is_available = network_policy_obj.is_exist(
+            resource_name=resource_name,
+        )
+
+        if not is_available:
+            # Set namespace value to the namespace where storageclient will be created
+            log.info(
+                "Updating namespace where to create storage client: %s",
+                namespace_to_create_storage_client,
+            )
+            network_policy_data["metadata"][
+                "namespace"
+            ] = namespace_to_create_storage_client
+
+            # Create network policy
+            network_policy_data_yaml = tempfile.NamedTemporaryFile(
+                mode="w+", prefix="network_policy", delete=False
+            )
+            templating.dump_data_to_temp_yaml(
+                network_policy_data, network_policy_data_yaml.name
+            )
+            log.info("Creating NetworkPolicy CR")
+            out = self.ocp_obj.exec_oc_cmd(f"apply -f {network_policy_data_yaml.name}")
+            log.info(f"output: {out}")
+            log.info(
+                f"Sleeping for 30 seconds after {network_policy_data_yaml.name} created"
+            )
+
+            assert network_policy_obj.check_resource_existence(
+                should_exist=True, timeout=300, resource_name=resource_name
+            ), log.error(
+                f"Networkpolicy does not exist for {namespace_to_create_storage_client} namespace"
+            )
+
+        else:
+            log.info(
+                f"Networkpolicy already exists for {namespace_to_create_storage_client} namespace"
+            )
+
+    def onboarding_token_generation_from_ui(
+        self,
+    ):
+        """
+        This method generates onboarding token from UI
+
+        Steps:
+        1:- Check private and public keys are available
+        2:- Check Storage-Clients pages available
+
+        Returns:
+        onboarding_token(str): client onboarding token
+
+        """
+        secret_ocp_obj = ocp.OCP(
+            kind=constants.SECRET, namespace=config.ENV_DATA["cluster_namespace"]
+        )
+        for secret_name in {
+            constants.ONBOARDING_PRIVATE_KEY,
+            constants.MANAGED_ONBOARDING_SECRET,
+        }:
+            assert secret_ocp_obj.is_exist(
+                resource_name=secret_name,
+            ), f"{secret_name} does not exist in {config.ENV_DATA['cluster_namespace']} namespace"
+
+        # verify storage-client page is available
+        onboarding_token = (
+            self.validation_ui_obj.verify_onboarding_token_generation_from_ui()
+        )
+        return onboarding_token
+
+    # @retry(AssertionError, 12, 10, 1)
+    # def create_storage_client(
```

Comment:
please remove all the commented code from the PR

Reply:
Addressed in latest commit.